cmd/system-probe/config/config.go

```diff
@@ -83,6 +83,9 @@ func newSysprobeConfig(configPath string, fleetPoliciesDirPath string) (*types.C
 	}
 
 	// Load the remote configuration
+	if fleetPoliciesDirPath == "" && os.Getenv("DD_FLEETCFGPATH") != "" {
+		fleetPoliciesDirPath = os.Getenv("DD_FLEETCFGPATH")
+	}
```

Comment:
No need to query the env twice

```suggestion
	if fleetPoliciesDirPath == "" {
		fleetPoliciesDirPath = os.Getenv("DD_FLEETCFGPATH")
	}
```